src/web/teap/requirements/dev.txt

```diff
@@ -0,0 +1,17 @@
+# Everything the developer needs in addition to the production requirements
```

Comment:
I don't like hardcoding those version numbers here in the repository, could you do a trick similar to nextcloud's `requirements.in`? Moreover, is `flake8` really a hard dependency?

Reply:
Ok, will refactor it to `requirements.in`. About `flake8` this is was by default in template, but only in development requirements, so won't be used in production, but still we can get rid of this anyways.

Reply:
Sure, it looks like that it comes from some kind of template. However, devs can use whatever they want for coding support, so let's keep this minimal. I dislike concerte versions outside of releases, as they require constant maintenance.